### DISCUSSION TITLE
About forward imaging.

### DISCUSSION THREAD

**Discussion by wanglinfeng1** [ucl-bug/helmnet#11] [Q&A] [answered]
From the Fig.12 and Fig.13 in the paper “A Helmholtz equation solver using unsupervised learning: Application to transcranial ultrasound” I find that most of the complex structure of the transverse CT slice in the field can be seen after just 200 iterations with this code. I am wondering can this code predict the sound field information at any given moment of a certain positionin in the brain? Or can I get a signal at this position that varies over time?

Thanks a lot for your suggestions.

**Comment by astanziola** [comment, marked as answer]
If your goal is to do something like a wave simulation, i.e. a time-stepping simulation, then the answer is no. Although it is true that the wavefield resembles a wave simulation as the solution propagates, you can also see later on that the wavefield pattern is static.
I believe is doing something similar to the [WaveHoltz](https://arxiv.org/abs/1910.10148) solver, although I'm not sure.

If your goal is to simulate the wave equation in the brain, I suggest you give a look at [this paper](https://asa.scitation.org/doi/10.1121/1.4976339) and derived works.

**Reply by wanglinfeng1** [reply]
I see what you mean and besides, thank you for your reply and useful advice.
